File: test/fixtures/csharp/get_with_data.cs
using System.Net.Http;

HttpClient client = new HttpClient();

HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Get, "http://localhost:28139/synthetics/api/v3/monitors?test=2&limit=100&w=4");

request.Headers.Add("X-Api-Key", "123456789");

HttpResponseMessage response = await client.SendAsync(request);
response.EnsureSuccessStatusCode();
string responseBody = await response.Content.ReadAsStringAsync();

File: test/fixtures/java/get_with_data.java
import java.io.IOException;
import java.net.URI;
import java.net.http.HttpClient;
import java.net.http.HttpRequest;
import java.net.http.HttpResponse;

HttpClient client = HttpClient.newHttpClient();

HttpRequest request = HttpRequest.newBuilder()
    .uri(URI.create("http://localhost:28139/synthetics/api/v3/monitors?test=2&limit=100&w=4"))
    .GET()
    .setHeader("X-Api-Key", "123456789")
    .build();

HttpResponse<String> response = client.send(request, HttpResponse.BodyHandlers.ofString());
